docs/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I'm not familiar with this file. What is it for? Any links for me to read up?

Reply:
https://realfavicongenerator.net/ I'm not sure what this is either. Got lazy and just C/P'd everything.